BitMLx_pipeline.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+<<COMMENT
+This script is used to pipelining all executions on files stored under "./output".
+
+The pipline: 
+    1. Run the Bitmlx compiler
+    2. Replace all hashes with python script
+    3. Compile each modified contract and store result in corresponding .blz files
+
+Usage: 
+    ./BitMLx_pipeline.sh
+COMMENT
+
+stack run || {
+    echo "Error: stack run failed."
+    exit 1
+}
+
+file_ext=".rkt"
+result_ext=".blz"
```

Comment:
For next PR: I just learned that the standard balzac extension is actually ".balzac", so we may want to change this and the python script to use that.

Reply:
Do you mean to change the result file extension into ".balzac"? I did it in my local branch, by changing the extension in the bash file, and everything is still working as before. It seems that one does not need to change something in the python script.